Readme.md

```diff
@@ -67,6 +67,10 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician.
 - [Muhammad Favian Revaldy](https://github.com/lukarukikato) [FAV]
   > As an academician.
+- [Mohammad Fiqri Firmansyah](https://github.com/TakanashaTaryu) [RYU]
+  > As an Academian 
+
+
```

Comment:
Remove these two newlines as its not needed here, thanks!

Reply:
done, I updated README.md